spec/models/course_spec.rb

```diff
@@ -819,6 +819,28 @@
         end
       end
     end
+
+    describe ".with_provider_name" do
```

Comment:
```suggestion
    describe "#with_provider_name" do
```

Reply:
Should note that this convention also appears elsewhere in the file, outside of this commits changes

Reply:
Sorry, deleted my old comment by mistake -  The other tests for similar scope have a ".", should those change as well? This site I randomly found thinks as it's a class method, rather than an instance method, it should be a "." http://www.betterspecs.org/

Reply:
I'd usually go with `.` for a class method and `#` for an instance method.